.travis.yml

```diff
@@ -36,4 +44,6 @@ install:
   - export PATH=$PWD/travis-phantomjs:$PATH
 
 script:
-  - ember try $EMBER_TRY_SCENARIO test
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
@JoshSmith - Ah! I did not read that correctly the first time. Surprised that isn't a bit more detailed in the comments above that line.